src/main/docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ LABEL ContainerName="${project.name}" ContainerSourceCode=${project.url}
 
 # Install packages and setup the application's user
 WORKDIR /opt/${project.artifactId}
-RUN apk add openjdk11 --quiet --no-cache \
+RUN apk add openjdk17 --quiet --no-cache \
```

Comment:
What about the other places where Java 11 is referenced (maven.compiler.release, etc.)?

Reply:
The parent project sets that dynamically depending on which JDK it finds on the system.

Reply:
I see, nevermind then, I was not aware of that